index.html

```diff
@@ -5,32 +5,22 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link rel="shortcut icon" type="image/png" href="favicon-16x16.png">
   <link rel="stylesheet" href="./css/style.css">
   <script defer src="./js/app.js"></script>
+  
 </head>
 <body>
-  <section>
-    <h1 class="animate__animated animate__bounce" id="title">TURN YOUR DESTINY</h1>
-    <div class="machine">
-        <div data-cell-index="0" class="cell"></div>
-        <div data-cell-index="1" class="cell"></div>
-        <div data-cell-index="2" class="cell"></div>
-        <div data-cell-index="3" class="cell"></div>
-        <div data-cell-index="4" class="cell"></div>
-        <div data-cell-index="5" class="cell"></div>
-        <div data-cell-index="6" class="cell"></div>
-        <div data-cell-index="7" class="cell"></div>
-        <div data-cell-index="8" class="cell"></div>
-        <div data-cell-index="9" class="cell"></div>
-        <div data-cell-index="10" class="cell"></div>
-        <div data-cell-index="11" class="cell"></div>
-        <div data-cell-index="12" class="cell"></div>
-        <div data-cell-index="13" class="cell"></div>
-        <div data-cell-index="14" class="cell"></div>
-        <div data-cell-index="15" class="cell"></div>
-  </div>
-    <h2 class="game--status"></h2>
-    <button class="spin">Spin Your Numbers</button>
-</section>
-</body>
+  <h1 >MATCH ALL & WIN 1k
+  </h1>
+  <h2 id = 'win'></h2>
+<div id = slot>
+  <div id = item1 class = item>🤑</div>
+  <div id = item2 class = item>💎</div>
+  <div id = item3 class = item>⚽</div>
```

Comment:
It'd also make more sense to call these `slot1`, `slot2`, and `slot3` since that most closely aligns with what they are - item it somewhat abstract and could be confusing.